sources/packages/forms/src/form-definitions/educationprogramoffering.json

```diff
@@ -2186,8 +2186,9 @@
           "allowCalculateOverride": false,
           "validate": {
             "required": true,
+            "onlyAvailableItems": false,

```

Comment:
Since the values are present in the form this value must be true.

Reply:
Not part of my change, but will do it. This occurs as these forms are created before the formio upgrade